lib/game_stats.rb

```diff
@@ -1,4 +1,94 @@
 
 
+
+
 module GameStats
+
+	def highest_total_score(season) # integer
+		if away_winners(season) > home_winners(season)
+			away_winners(season)
+		elsif away_winners(season) < home_winners(season)
+			home_winners(season)
+		end
+	end
+
+	def lowest_total_score(season)
+		if away_winners(season) < home_winners(season)
```

Comment:
Ok, so unfortunately I think you may have coded for the wrong stat on highest_total_score and lowest_total_score. Not sure if you saw Austin K's question to Kat in code-help a few days ago. But these methods should be looking at every game and determining the sums of home goals and away goals (if the score was 3-1, the "total score" would be 4). We can assume lowest_total_score would probably return 0 (from a 0-0 tie), but highest total score would probably be an integer around 10 or so

Reply:
Good call, also thanks for explaining this to me over zoom. I totally misinterpreted those methods when reading the directions but will rework them today to get them working the way they are supposed to.